### PR TITLE
corrected the grammar of option descriptions

### PR DESCRIPTION
**Pull request by m-ahmadi**
corrected some grammar typos of commander option descriptions. (punctuation/capitalization/etc)

### CODE REVIEW

File: packages/babel-cli/src/babel/options.js [hunk 1]
```diff
@@ -40,114 +40,114 @@ commander.option(
...
   "--source-root [filename]",
-  "the root from which all sources are relative",
+  "The root from which all sources are relative.",
 );
 
 // Config params for certain module output formats.
 commander.option(
   "--module-root [filename]",
-  "optional prefix for the AMD module formatter that will be prepend to the filename on module definitions",
+  "Optional prefix for the AMD module formatter that will be prepended to the filename on module definitions.",
```
Comment (JLHwung) [inline packages/babel-cli/src/babel/options.js]: Could you add `// eslint-disable-next-line max-len` above this line?

**Comment by JLHwung** [comment]
CI failure has been fixed on master. Thus it is not relevant.

File: packages/babel-cli/src/babel/options.js [hunk 2]
```diff
@@ -40,114 +40,115 @@ commander.option(
...
-  "retain line numbers - will result in really ugly code",
+  "Retain line numbers. This will result in really ugly code.",
 );
 commander.option(
   "--compact [true|false|auto]",
-  "do not include superfluous whitespace characters and line terminators",
+  "Do not include superfluous whitespace characters and line terminators.",
   booleanify,
 );
 commander.option("--minified", "save as much bytes when printing [true|false]");
```
Comment (m-ahmadi) [inline packages/babel-cli/src/babel/options.js]: Dang it, I missed this line.
@JLHwung Could you replace this description with `"Save as many bytes when printing [true|false]"` in a future commit?

Comment (JLHwung) [reply packages/babel-cli/src/babel/options.js]: Ah, could you submit another PR?

Comment (m-ahmadi) [reply packages/babel-cli/src/babel/options.js]: OK.

**Pull request merged**